src/test/java/uk/gov/pay/connector/gateway/worldpay/wallets/WorldpayWalletAuthorisationHandlerTest.java

```diff
@@ -110,16 +111,41 @@ public void shouldSendGooglePay3DSRequestWhenGooglePayDetailsWithoutIpAddressAre
         }
     }
 
+    @Test
+    public void shouldSendGooglePay3DSRequestWhenGooglePayDetailsWithIpAddressArePresent() throws Exception {
+        try {
+            worldpayApplePayAuthorisationHandler.authorise(getGooglePay3DSAuthorisationRequest(true, true));
```

Comment:
Can we remove `Apple`? We are handling Google Pay 3DS authorisation request.
```suggestion
            worldpayPayAuthorisationHandler.authorise(getGooglePay3DSAuthorisationRequest(true, true));
```

Reply:
Done.